config/initializers/sidekiq.rb

```diff
@@ -8,3 +8,11 @@
 Sidekiq.configure_server do |config|
   config.redis = Sidekiq::RedisConnection.create(namespace: Rails.env, url: redis_url)
 end
+
+ActiveSupport.on_load(:active_record) do
+  unless Sidekiq::Cron::Job.find('Reminders')
+    Sidekiq::Cron::Job.create(
```

Comment:
Thoughts on only creating the cron task in certain environments (production only)?

Reply:
That seems fine, I'll update it.